src/hipsycl_rewrite_includes/InclusionRewriter.cpp

```diff
@@ -195,6 +201,9 @@ void InclusionRewriter::FileSkipped(const FileEntry &/*SkippedFile*/,
                                     SrcMgr::CharacteristicKind /*FileType*/) {
   assert(LastInclusionLocation.isValid() &&
          "A file, that wasn't found via an inclusion directive, was skipped");
+  auto P = SkippedIncludes.insert(LastInclusionLocation.getRawEncoding());
+  (void)P;
```

Comment:
What does this do? `P` is just an `std::pair`, right?

Reply:
I copied this from the implementation of `FileChanged`, it's most likely just to prevent a compiler warning about `P` being unused during release builds (when `assert` is a no-op).

Reply:
Okay, then let's keep it that way. Perhaps you could add a comment?

Reply:
Okay, that's fine :)